mlflow/exceptions.py

```diff
@@ -37,7 +37,7 @@ def serialize_as_json(self):
 class RestException(MlflowException):
     """Exception thrown on non 200-level responses from the REST API"""
     def __init__(self, json):
-        error_code = json['error_code']
+        error_code = json.get('error_code')
         message = error_code
         if 'message' in json:
             message = "%s: %s" % (error_code, json['message'])
```

Comment:
In this case, the message could be `None: <actual message>`. Maybe we could default to INTERNAL_ERROR instead?